### DISCUSSION TITLE
startOnStep() in HasWizard page

### DISCUSSION THREAD

**Discussion by Grazulex** [filamentphp/filament#3821] [Help] [answered]
Hello,

I have made a resource action page with the HasWizard trait and everything is perfect.
The only small problem is that I don't know how I can use the StartOnStep method in the getSteps function.... In a wizard field is easy but here is not so easy to do it

Can somebody help me ?

JMS

**Comment by danharrin** [comment, marked as answer]
Just added new methods that you can use to do this: #3824
